src/_pytest/logging.py

```diff
@@ -321,15 +321,19 @@ def __init__(self, item) -> None:
         """Creates a new funcarg."""
         self._item = item
         # dict of log name -> log level
-        self._initial_log_levels = {}  # type: Dict[str, int]
+        self._initial_logger_levels = {}  # type: Dict[str, int]
+        self._initial_handler_level = None  # type: Optional[int]
```

Comment:
I think it is not needed to reset the handler level, because each test gets a new instance of a LogCaptureHandler.

Reply:
Sry I was wrong. It it indeed needed if the `at_level` ctx manager is used. However, in the `_finalize`  method it is not needed.

Reply:
It's not used in the context manager though? If it's not needed in `_finalize` then surely it's not needed anywhere?

Reply:
You are right. Then it is not needed anywhere.

Reply:
Removed